mappings/net/minecraft/item/ItemGroups.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/unmapped/C_debvfuic net/minecraft/item/ItemGroups
+	FIELD f_bmtsybyi SURVIVAL_INVENTORY_GROUP Lnet/minecraft/unmapped/C_lnokcayd;
```

Comment:
Why is this "survival inventory"? Shouldn't it be just "inventory"?

Reply:
That's the full name of the tab on the English localization; And plus, only INVENTORY can be a bit ambiguous